Formula/mafft.rb

```diff
@@ -0,0 +1,43 @@
+class Mafft < Formula
+  desc "Multiple alignments with fast Fourier transforms"
+  homepage "https://mafft.cbrc.jp/alignment/software/"
+  url "https://mafft.cbrc.jp/alignment/software/mafft-7.313-with-extensions-src.tgz"
+  sha256 "c48e5e05b427cae0d862daaef6148675d5ef57e24425c17b4c3d8da5b060eabd"
+
+  fails_with :clang do
+    build 421
+    cause <<-EOS
+      Clang does not allow default arguments in out-of-line definitions of
+      class template members.
+      EOS
+  end
+
+  def install
+    make_args = %W[CC=#{ENV.cc} CXX=#{ENV.cxx} CFAGS=#{ENV.cflags}
+                   CXXFLAGS=#{ENV.cxxflags} PREFIX=#{prefix} MANDIR=#{man1}]
+    make_args << "ENABLE_MULTITHREAD=" if MacOS.version <= :snow_leopard
```

Comment:
Let's remove the Snow Leopard case and caveat, and add `depends_on :macos => :lion`